ckan/controllers/group.py

```diff
@@ -295,15 +305,17 @@ def pager_url(q=None, page=None):
                     facets[facet] = facet
 
             # Facet titles
-            for plugin in plugins.PluginImplementations(plugins.IFacets):
-                if self.group_type == 'organization':
+            from ckan.controllers.organization import OrganizationController
+            if isinstance(self, OrganizationController):
```

Comment:
then we have three ways testing if this is related to an organization. that doesn't seem good.